erdpy/projects/eei_activation.py

```diff
@@ -0,0 +1,39 @@
+
+import logging
+import sys
+
+import requests
+import toml
+from erdpy.diskcache import DiskCache
+from erdpy.proxy.core import ElrondProxy
+
+logger = logging.getLogger("eei")
+
+
+class ActivationKnowledge(DiskCache):
```

Comment:
Optionally, consider renaming to `ActivationEpochsInfo`, it might be more obvious.

Reply:
Renamed.